src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -34,7 +34,7 @@
     private transient AuthorConfiguration authorConfig;
     private transient boolean isStandaloneConfigIgnored;
     private transient List<CommitHash> ignoreCommitList;
-    private transient boolean isFormatsOverriding;
+    private transient boolean isCsvConfigOverridingFormats;
```

Comment:
This naming is weird. Did you consider the situation where we pass in formats through Command Line?

Reply:
> Did you consider the situation where we pass in formats through Command Line?

Yes, but they don't set the variable `isFormatsOverriding` to true. The only case where `isFormatsOverriding` is set to true is when you are using CSV config files and that you set the column in `File Formats` in `repo-config.csv` to `override:java;md;...`

Meanwhile I'll change the variable name back to `isFormatsOverriding` and see how we will proceed with this

Reply:
OK.